suite-native/transactions/src/components/transactionDetail/TransactionDetailData.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { Box } from '@suite-native/atoms';
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+
+import { TransactionDetailDataRow } from './TransactionDetailDataRow';
+
+type TransactionDetailDataProps = {
+    transaction: WalletAccountTransaction;
+};
+
+export const TransactionDetailData = ({ transaction }: TransactionDetailDataProps) => {
+    const getBlockTime = () => {
+        if (!transaction.blockTime) return '';
```

Comment:
Are we going to use this transaction _blockTime_ somewhere else in the app? I would say yes so this could be moved to some util function or even better - formatter :). But formatters are still in progress...

Reply:
Yeah this is just temporary solution. When formatters are ready, this will definitely be used as a selector, where you pass txid and it will return transaction block time already formatted.